.github/workflows/CI.yml

```diff
@@ -0,0 +1,22 @@
+name: Check code format
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: install clang-format
+        run: |
+          sudo apt update
+          sudo apt install -y clang-format
+      - name: checkout repository content
+        uses: actions/checkout@v3
+      - name: setup python
+        uses: actions/setup-python@v3
+      - name: run python Script
+        run: ./script/run-clang-format.py -rq .
```

Comment:
**suggestion:** We must keep the CI green at all times (ideally) :) Could we mark this job `continue-on-error: true` so the failure is skipped and does not reflect on the overall pipeline result 🤔 ?

Reply:
> **suggestion:** We must keep the CI green at all times (ideally) :) Could we mark this job `continue-on-error: true` so the failure is skipped and does not reflect on the overall pipeline result 🤔 ?

Yes we can, but I think it's better to keep the current approach. The CI will pass (green) if all files follow the format config.

Or you want to have this approach: CI passes anyway but output the files/code need to be formatted?